setup.py

```diff
@@ -19,6 +13,6 @@
     author_email='developers@protecht.io',
     url='https://github.com/TicketGuardian/ticketguardian-python',
     keywords=['sdk', 'TicketGuardian', 'python'],
-    install_requires=requirements,
+    install_requires=['PyJWT==1.5.3', 'simplejson==3.16.0', 'requests[security]'],
```

Comment:
Why are we switching to hardcoding the reqs?

Reply:
Sorry I should have left a note on this. I was unable to get `tox` working correctly with the existing code to get `install_requires` so I hard coded in the values it would have been using here instead.